test/empower_test.js

```diff
@@ -513,4 +517,63 @@ suite('onSuccess can throw', function () {
     });
 });
 
+suite('additionalMethods', function () {
+    var assert = empower(
+      {
+          fail: function (message) {
+              baseAssert.ok(false, message);
+          },
+          pass: function (message) {
+              // noop
+          }
+      },
+      {
+          additionalMethods: [
+              'assert.fail([message])',
+              'assert.pass([message])'
+          ],
+          onError: function (event) {
+              baseAssert.equal(event.type, 'error');
+              return event;
+          },
+          onSuccess: function (event) {
+              baseAssert.equal(event.type, 'success');
+              return event;
+          }
+      }
+    );
+
+    test('instrumented code: success', function () {
+        var event = eval(weave('assert.pass("woot!")'));
+        baseAssert.equal(event.type, 'success');
+        baseAssert.strictEqual(event.enhanced, false);
+        baseAssert.equal(event.originalMessage, 'woot!');
```

Comment:
We extract "originalMessage" even though it passed. Your test runner can then log assertions easily:
```js
function onSuccess(event) {
  console.log('passed: ' + originalMessage);
}
```

It decouples your assertions, and knowledge about which argument is the message argument from your logger.